.github/workflows/ruff.yml

```
@@ -10,11 +10,46 @@ jobs:
-          src: wyoming_whisper_trt/ tests/
-          args: format --check
+          version: "0.15.10"
+          src: "."
+          args: "format"
+
+      - name: Commit auto-fixes
+        if: github.event.pull_request.head.repo.full_name == github.repository
+        shell: bash
+        env:
+          CI_PUSH_TOKEN: ${{ secrets.CI_PUSH_TOKEN }}
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)